README.md

```diff
@@ -149,7 +149,7 @@ import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 import scala.scalanative.sbtplugin.ScalaNativePlugin.autoImport._
 import sbtcrossproject.CrossPlugin.autoImport._
 import scalajscrossproject.ScalaJSCrossPlugin.autoImport.{toScalaJSGroupID => _, _}
-import scalajscrossproject.JSPlatform
+import org.scalajs.sbtplugin.JSPlatform
```

Comment:
Actually, this should be `scalajscrossproject.ScalaJSCrossPlugin.autoImport.JSPlatform`.

The fact that it is still accessible as `org.scalajs.sbtplugin.JSPlatform` is a mistake. That one should be deprecated.

Reply:
Filed as #83.